docs/_docs/quickstart.md

```diff
@@ -4,25 +4,37 @@ title: Quick-start guide
 
 ```sh
+# Install Jekyll and Bundler gems through RubyGems
 ~ $ gem install jekyll bundler
+
+# Create a new Jekyll site at ./myblog
 ~ $ jekyll new myblog
+
+# Change into your new directory
 ~ $ cd myblog
+
+# Build the site on the preview server
 ~/myblog $ bundle exec jekyll serve
-# => Now browse to http://localhost:4000
+
+# Now browse to http://localhost:4000
 ```
 
-The `jekyll new` command now automatically initiates `bundle install` and installs the dependencies required. To skip this, pass `--skip-bundle` option like so `jekyll new myblog --skip-bundle`.
+`gem install jekyll bundler` installs the [jekyll](https://rubygems.org/gems/jekyll/) and [bundler](https://rubygems.org/gems/bundler) gems through [RubyGems](https://rubygems.org/). You need only to install the gems one time &mdash; not every time you create a new Jekyll project. Here are some additional details:
+
+* `bundler` is a gem that manages other Ruby gems. It makes sure your gems and gem versions are compatible, and that you have all necessary dependencies each gem requires.
+* The `Gemfile` and `Gemfile.lock` files inform Bundler about the gem requirements in your theme. If your theme doesn't have these Gemfiles, you can omit `bundle exec` and just run `jekyll serve`.
+
+* When you run `bundle exec jekyll serve`, Bundler uses the gems and versions as specified in `Gemfile.lock` to ensure your Jekyll site builds with no compatibility or dependency conflicts.
 
-If you wish to install jekyll into an existing directory, you can do so by running `jekyll new .` from within the directory instead of creating a new one. If the existing directory isn't empty, you'll also have to pass the `--force` option like so `jekyll new . --force`.
+`jekyll new <PATH>` installs a new Jekyll site at the path specified (relative to current directory). In this case, Jekyll will be installed in a directory called `myblog`. Here are some additional details:
 
-That's nothing, though. The real magic happens when you start creating blog
-posts, using the front matter to control templates and layouts, and taking
-advantage of all the awesome configuration options Jekyll makes available.
+* To install the Jekyll site into the directory you're currently in, run `jekyll new .` If the existing directory isn't empty, you can pass the `--force` option with `jekyll new . --force`.  
+* `jekyll new` automatically initiates `bundle install` to install the dependencies required. (If you don't want Bundler to install the gems, use `jekyll new myblog --skip-bundle`.)
+* By default, Jekyll installs a gem-based theme called [Minima](https://github.com/jekyll/minima). With gem-based themes, some of the theme directories and files are stored in the gem, hidden from view in your Jekyll project. To better understand themes, see [Themes](../themes).
```

Comment:
updated. i think the subheadings and bullets facilitate scanning enough. the code sample is at the top, so users don't have to read beyond that code sample to get things running. but i wouldn't recommend reducing explanations of the code in the goal of making it seem simpler. we run the risk of making the code too cryptic. 

if anything, i'd cut the bullets about skipping bundler or forcing jekyll to install in a folder that already has content in it. i don't see much use for those two options, but i think it's okay to have them there.